aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageUploadFileOperation.java

```diff
@@ -185,11 +185,7 @@ public void onStateChanged(int transferId, TransferState state) {
                     onSuccess.accept(StorageUploadFileResult.fromKey(getRequest().getKey()));
                     return;
                 case FAILED:
-                    onError.accept(new StorageException(
-                            "Storage upload operation was interrupted.",
-                            "Please verify that you have a stable internet connection."
-                    ));
-                    return;
+                    // no-op;
```

Comment:
*Note:* As above, `onError.accept` is already invoked on line 207 below

https://github.com/aws-amplify/amplify-android/blob/b7c442e92c722c2b853d2efc6c79143859a4ad62/aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageUploadFileOperation.java#L203-L212